backend/curriculum_tracking/api_views.py

```diff
@@ -572,3 +572,30 @@ def assign_user_as_reviewer(self, request, pk=None):
 
     # TODO: bulk set due dates
 
+
+class WorkshopAttendanceViewset(AuthMixin, viewsets.ModelViewSet):
+
+    serializer_class = serializers.WorkshopAttendanceSerializer
+
+    filter_backends = [DjangoFilterBackend]
+    filterset_fields = ["attendee_user"]
+
+    def get_permissions(self):
+        if self.action == "retrieve":
+            permission_classes = [
+                curriculum_permissions.IsProjectAssignee
```

Comment:
Are you sure that these permissions are correct? 

IsProjectAssignee checks that the current logged in user is equal to the project.recruit_user.
There is no project here, and WorkshopAttendance doesn't have a `recruit_user` field.

Try GET an instance via the api. You'll get a 500 I think.

Never guess! And take names seriously, if something says it is for projects and you try to apply it to a thing that has nothing to do with projects then it probably wont work.